src/coders/byte-stream.js

```diff
@@ -59,6 +117,14 @@ class ByteStream {
         return st;
     }
 
+    /**
+     * Write bytes from the given Uint8Array array to the stream and increment
+     * the position.
+     * @param {Uint8Array} bytes - Bytes to write to the stream.
+     * @param {number=} [start=0] - Where in bytes to start writing from.
+     * @param {nubmmer=} [end=bytes.length] - Where in bytes to stop writing.
```

Comment:
Maybe add whether bytes[end] will be written to or not. It looks like not..

Also there is a typo: `nubmmer` => `number`

```suggestion
     * @param {number=} [end=bytes.length] - Where in bytes to stop writing, excluding position at bytes[end].
```